### PR TITLE
fix: disable cgroup metrics for Kepler

### PR DESCRIPTION
**Pull request by vprashar2929**
Sets `EXPOSE_CGROUP_METRICS=false` in Kepler's config. This change addresses issue #399

### CODE REVIEW

File: pkg/components/exporter/exporter.go [hunk 1]
```diff
@@ -257,7 +257,7 @@ func NewConfigMap(d components.Detail, k *v1alpha1.KeplerInternal) *corev1.Confi
 		"EXPOSE_HW_COUNTER_METRICS":  "true",
 		"EXPOSE_IRQ_COUNTER_METRICS": "true",
 		"EXPOSE_KUBELET_METRICS":     "true",
-		"EXPOSE_CGROUP_METRICS":      "true",
+		"EXPOSE_CGROUP_METRICS":      "false",
```
Comment (sthaha) [inline pkg/components/exporter/exporter.go:260]: https://github.com/sustainable-computing-io/kepler-operator/blob/v1alpha1/pkg/components/exporter/exporter.go#L620 Ahem ahem :p

**Review by sthaha — CHANGES_REQUESTED** [review]
more than one place to fix :)

File: pkg/components/exporter/exporter.go (hunk 1 above)
Comment (sthaha) [reply pkg/components/exporter/exporter.go:260]: Also, I think we can completely remove this from the configmap
cc: @marceloamaral ?

Comment (vprashar2929) [reply pkg/components/exporter/exporter.go:260]: I think by default Kepler takes value of `ENABLE_EBPF_CGROUPID` as [true](https://github.com/sustainable-computing-io/kepler/blob/29d0b00f2362ac28d14b887af9a3eb6140f281b0/pkg/config/config.go#L70) and `EXPOSE_CGROUP_METRICS` as [true](https://github.com/sustainable-computing-io/kepler/blob/29d0b00f2362ac28d14b887af9a3eb6140f281b0/pkg/config/config.go#L77)

File: pkg/components/exporter/exporter.go [hunk 2]
```diff
@@ -617,7 +617,7 @@ func newExporterContainer(kiName, dsName string, deployment v1alpha1.InternalExp
 		Command: []string{
 			"/usr/bin/kepler",
 			"-address", bindAddress,
-			"-enable-cgroup-id=true",
+			"-enable-cgroup-id=$(ENABLE_EBPF_CGROUPID)",
 			"-enable-gpu=$(ENABLE_GPU)",
```
Comment (sthaha) [inline pkg/components/exporter/exporter.go:621]: can we not remove both since the config already has this value?

**Review by sthaha — CHANGES_REQUESTED** [review]

**Review by sthaha — APPROVED** [review]

**Pull request merged**